File: src/main/java/com/zjnh/java/io/IO.java
package com.zjnh.java.io;

import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;

public class IO {
    public static void main(String... args) {
        BufferedReader bufferedReader = null;
        try {
            bufferedReader = new BufferedReader(new FileReader(new File("build.gradle.kts")));
            String line;
            while ((line = bufferedReader.readLine()) != null) {
                System.out.println(line);
            }
        } catch (IOException e) {
            e.printStackTrace();
        } finally {
            try {
                if (bufferedReader != null) {
                    bufferedReader.close();
                }
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
    }

}

File: src/main/kotlin/com/zjnh/kotlin/io/IO.kt
package com.zjnh.kotlin.io

import java.io.BufferedReader
import java.io.File
import java.io.FileReader

fun main(args: Array<String>) {
    val file = File("build.gradle.kts")
    val bufferedReader = BufferedReader(FileReader(file))
    var line: String
    while (true) {
        line = bufferedReader.readLine() ?: break
        println(line)
    }
    bufferedReader.close()
}